lib/config.js

```diff
@@ -45,6 +45,14 @@ export default function subscribeConfig(conn, configChanged) {
       updateConfig({ services: Object.assign({}, config.services, { [domain]: domainInfo }) });
     }
 
+    function processServiceRemoved(event) {
+      if (config === null) return;
+
+      const { domain, service } = event.data;
+
+      updateConfig({ services: Object.assign({}, config.services, Object.assign({}, config.services.[domain], { [service]: null} )) });
```

Comment:
`config.services.[domain]` <- That's not valid JS.

Setting the service to `null` will break things too as people check for existence of key in this object. Instead, create your own copy object code that will skip the key.

```js
if (!(domain in config.services)) return;

const curDomainInfo = config.services[domain];

if (!curDomainInfo || !(service in curDomainInfo)) return;

const domainInfo = {};
Object.keys(curDomainInfo).forEach((sKey) => {
  if (sKey !== service) domainInfo[sKey] = curDomainInfo[sKey];
});
updateConfig({ services: Object.assign({}, config.services, { [domain]: domainInfo }) });
```